SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -460,6 +452,24 @@ bool ValidateAndCreateSnapshotGenerationPath(FString& SavePath)
 	return true;
 }
 
+
+bool RunUserSnapshotGenerationOverrides(Worker_SnapshotOutputStream* OutputStream) {
+	Worker_EntityId NextEntityId = SpatialConstants::PLACEHOLDER_ENTITY_ID_LAST + 1;
```

Comment:
can this be const?

Reply:
I don't think so, it's passed as a reference to user who should be incrementing it?